README.md

```diff
@@ -65,13 +65,32 @@ You can use that for all kinds of customization.
 Add extra settings in the call to `React.createElement(GraphiQL, {})` in the published view:
 
 ```js
-React.createElement(GraphiQL, {
-  fetcher: GraphiQL.createFetcher({
-    url: "{{ $url }}",
-    subscriptionUrl: "{{ $subscriptionUrl }}",
-  }),
-  // See https://github.com/graphql/graphiql/tree/main/packages/graphiql#props for available settings
-});
+function GraphiQLWithExplorer() {
+    const [query, setQuery] = React.useState('');
+    const fetcher = GraphiQL.createFetcher({
+        url: '{{ $url }}',
+        subscriptionUrl: '{{ $subscriptionUrl }}',
+    })
```

Comment:
Why did you move this inside `function GraphiQLWithExplorer()`? That will cause it to be recreated on every render, which seems wasteful.

Reply:
I have updated the docs to remove duplication.